tests/unit/frontend/forms/helpers/CsvMergedHeadersGenerator.test.js

```diff
@@ -173,30 +165,6 @@ describe('CsvMergedHeadersGenerator', () => {
       expect(Object.values(generator.fieldIdToNumCols)).toEqual([2])
     })
 
-    it('should handle adding of single header record', () => {
```

Comment:
This test no longer passes with the new implementation because we can't instantiate a `Response` object, and any object that doesn't fit the three response types will result in an error. I'm not completely sure and do correct me if I'm wrong, but I don't think there's a case where we will try to do this?

Reply:
I'm pretty sure this is important because it seems to be checking for the case where a form contains a single statement field. @karrui can correct me if I'm wrong. does the code work for that case?

Reply:
I tested and it still works. I printed the record for statement fields and the `answer` has value `Success` so a `SingleAnswerResponse` object will still be created

Reply:
we shouldn't remove this test case. Why would it fail? If it fails does it not mean the code is not working? This test is checking that indeed fields with `isHeader = true` does not get added to the `fieldIdToQuestion` map